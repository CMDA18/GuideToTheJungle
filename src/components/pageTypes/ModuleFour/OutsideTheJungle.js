// @flow
import React, { Component } from 'react'
import Helmet from 'react-helmet'
// import styled from 'styled-components'
import Grid from '../../Grid'
import PageTitle from '../../PageTitle'
import Intro from '../../Intro'
import { outsideTheJungle } from '../../../constants/content/index'

type Props = {
  test: string
}

class OutsideTheJungle extends Component<Props> {
  render () {
    return (
      <div className='outsideTheJungle' data-test={this.props.test}>
        <Helmet
          title='outsideTheJungle'
        />
        <Grid page='howWeWork'>
          <header>
            <PageTitle {...outsideTheJungle}/>
            <Intro {...outsideTheJungle}/>
          </header>
        </Grid>
      </div>
    )
  };
}

export default OutsideTheJungle
