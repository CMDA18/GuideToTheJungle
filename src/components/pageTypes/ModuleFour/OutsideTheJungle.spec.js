import React from 'react'
import { shallow } from 'enzyme'
import OutsideTheJungle from './OutsideTheJungle'

it('renders without crashing', () => {
  expect(shallow(<OutsideTheJungle />)).toMatchSnapshot()
})
