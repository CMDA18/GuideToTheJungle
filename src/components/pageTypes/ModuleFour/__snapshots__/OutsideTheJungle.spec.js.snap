// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OutsideTheJungle />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          title="outsideTheJungle"
        />,
        <Grid
          page="howWeWork"
        >
          <header>
            <PageTitle
              introText="test"
              introTitle="Transparantie"
              pageTitle="4.3 Buiten de Jungle"
            />
            <Intro
              introText="test"
              introTitle="Transparantie"
              pageTitle="4.3 Buiten de Jungle"
            />
          </header>
        </Grid>,
      ],
      "className": "outsideTheJungle",
      "data-test": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "title": "outsideTheJungle",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <header>
            <PageTitle
              introText="test"
              introTitle="Transparantie"
              pageTitle="4.3 Buiten de Jungle"
            />
            <Intro
              introText="test"
              introTitle="Transparantie"
              pageTitle="4.3 Buiten de Jungle"
            />
          </header>,
          "page": "howWeWork",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <PageTitle
                introText="test"
                introTitle="Transparantie"
                pageTitle="4.3 Buiten de Jungle"
              />,
              <Intro
                introText="test"
                introTitle="Transparantie"
                pageTitle="4.3 Buiten de Jungle"
              />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "introText": "test",
                "introTitle": "Transparantie",
                "pageTitle": "4.3 Buiten de Jungle",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "introText": "test",
                "introTitle": "Transparantie",
                "pageTitle": "4.3 Buiten de Jungle",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "header",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            title="outsideTheJungle"
          />,
          <Grid
            page="howWeWork"
          >
            <header>
              <PageTitle
                introText="test"
                introTitle="Transparantie"
                pageTitle="4.3 Buiten de Jungle"
              />
              <Intro
                introText="test"
                introTitle="Transparantie"
                pageTitle="4.3 Buiten de Jungle"
              />
            </header>
          </Grid>,
        ],
        "className": "outsideTheJungle",
        "data-test": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "title": "outsideTheJungle",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <header>
              <PageTitle
                introText="test"
                introTitle="Transparantie"
                pageTitle="4.3 Buiten de Jungle"
              />
              <Intro
                introText="test"
                introTitle="Transparantie"
                pageTitle="4.3 Buiten de Jungle"
              />
            </header>,
            "page": "howWeWork",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <PageTitle
                  introText="test"
                  introTitle="Transparantie"
                  pageTitle="4.3 Buiten de Jungle"
                />,
                <Intro
                  introText="test"
                  introTitle="Transparantie"
                  pageTitle="4.3 Buiten de Jungle"
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "introText": "test",
                  "introTitle": "Transparantie",
                  "pageTitle": "4.3 Buiten de Jungle",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "introText": "test",
                  "introTitle": "Transparantie",
                  "pageTitle": "4.3 Buiten de Jungle",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "header",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
